modules/unsupported/ysld/src/main/java/org/geotools/ysld/encode/YsldEncodeHandler.java

```diff
@@ -153,72 +156,73 @@ public void remove() {
         return this;
     }
 
-    YsldEncodeHandler<T> inline(Map<String,Object> values) {
+    YsldEncodeHandler<T> inline(Map<String, Object> values) {
         stack.peek().putAll(values);
         return this;
     }
-    
+
     Object toColorOrNull(Expression expr) {
         Object obj = toObjOrNull(expr, false);
         if (obj instanceof String && expr instanceof Literal) {
             String str = Util.stripQuotes(obj.toString());
-            obj=makeColorIfPossible(str);
+            obj = makeColorIfPossible(str);
         }
         return obj;
     }
-    
+
     Object toObjOrNull(Expression expr) {
         return toObjOrNull(expr, false);
     }
-    
-    static final Pattern COLOR_PATTERN =  Pattern.compile("^#?([a-fA-F0-9]{6})$");
+
+    static final Pattern COLOR_PATTERN = Pattern.compile("^#?([a-fA-F0-9]{6})$");
+
     Object makeColorIfPossible(String str) {
         Matcher m = COLOR_PATTERN.matcher(str);
-        if(m.matches()) {
+        if (m.matches()) {
             // If it matches the regexp, then we know it should parse
             int i = Integer.parseInt(m.group(1), 16);
             return new Color(i);
         } else {
             return str;
         }
     }
-    
+
     static final Pattern EMBEDED_EXPRESSION_TO_ESCAPE = Pattern.compile("[$}\\\\]");
+
     String escapeForEmbededCQL(String s) {
         return EMBEDED_EXPRESSION_TO_ESCAPE.matcher(s).replaceAll("\\\\$0");
     }
-    
+
     Object toObjOrNull(Expression expr, boolean isname) {
```

Comment:
This method seems to be the crux of this class, and due to the complicated escaping nature of embeded ECQL statements could use a javadoc.